packages/vue-client/src/utils/format-utils.ts

```diff
@@ -0,0 +1,7 @@
+export function toUpperCaseFirstLetter(string: string) {
```

Comment:
Aside: We should really have some UI-friendly strings defined. I feel like the variant names are not designed to be shown to the user at all.